modules/utils/2.0/config/default.yaml

```diff
@@ -0,0 +1,31 @@
+## Lines commented out with `#!` are required for the module to run
+## Lines commented out with `#?` can optionally be user-configured
+## Lines commented out with `##` act as regular comments
+
+lcr-modules:
+    utils:
+        inputs:
+            bed: "__UPDATE__"
```

Comment:
I'm not sure what this BED file is supposed to be. We might have to use a more specific name.

Reply:
this is to create the interval list used in picard_qc because we talked about providing the bed files and write the rule in the utils module?